docs/topics/templates/document-attributes.adoc

```diff
@@ -83,7 +83,7 @@
 :ProductDocCoreGuideURL: https://github.com/windup/windup/wiki/Core-Development-Guide
 
 // URL for downloads on developers.redhat.com
-:ProductDownloadURL: https://developers.redhat.com/products/
+:ProductDownloadURL: https://developers.redhat.com/download-manager/file/
```

Comment:
Direct links are way too much work to maintain and very tricky. Sometimes a build does not get uploaded to the dev site in time, which means the link is temporarily broken.

I recommend that you change all the direct download links to this:

"Navigate to the `Migration Toolkit for Applications download` site[https://developers.redhat.com/products/mta/download] and download the ...."

At least that way, if the new build has not been uploaded, the user can bookmark the page or choose to download an older version of the file.